libs/dal/src/repositories/integration/integration.schema.ts

```diff
@@ -45,11 +45,14 @@ const integrationSchema = new Schema<IntegrationDBModel>(
       redirectUrl: Schema.Types.String,
       hmac: Schema.Types.Boolean,
     },
-
     active: {
       type: Schema.Types.Boolean,
       default: false,
     },
+    name: Schema.Types.String,
+    identifier: Schema.Types.String,
+    kind: Schema.Types.String,
+    kindId: Schema.Types.ObjectId,
```

Comment:
Should we already add conditions? But as an empty array for now? 
To make it easier for next phase

Reply:
Better wait for now. If we then change our mind we might forget to clean it.